src/ovp_pipeline/ui/view_models.py

```
@@ -4246,6 +4246,77 @@ def build_atlas_browser_payload(
+    *,
+    pack_name: str | None = None,
+    top_n: int | None = None,
+) -> dict[str, Any]:
+    from ..synthesis.curated_atlas import build_curated_atlas
+    from ..synthesis._shared import CRYSTAL_DIR_REL
+
+    requested_pack = pack_name or ""
+    pack = pack_name or PRIMARY_PACK_NAME
+    requested_top_n = top_n if top_n is not None else CURATED_ATLAS_DEFAULT_TOP_N
+    if requested_top_n < 1:
+        requested_top_n = 1
```

Ruff (0.15.12)

[warning] Replace `if` statement with `requested_top_n = max(requested_top_n, 1)`

Replace with `requested_top_n = max(requested_top_n, 1)`

(PLR1730)